README.md

```diff
@@ -95,7 +95,7 @@ be loaded by `pdf.js`. The PDF.js files are large and should be minified for pro
 ## Using PDF.js in a web application
 
 To use PDF.js in a web application you can choose to use a pre-built version of the library
-or to build it from source. We supply pre-built versions for usage with NPM and Bower under
+or to build it from the source. We supply pre-built versions for usage with NPM and Bower under
```

Comment:
"build from source" seem to be a very commonly used phrase, so I'm not sure that we need to change this!?

Reply:
It just sounded better in my head :)
Should I remove this?

Reply:
I'll remove it.

Reply:
I'm going to close this and make a new PR, just to simplify things.